alias.txt

```diff
@@ -146,56 +178,97 @@
 .dleave Verlassen Sie die Kontrollzone über $uc($1).
 .denter Fliegen Sie in die Kontrollzone über $uc($1).
+.dnoctr Bleiben Sie außerhalb der Kontrollzone.
+.dnod Bleiben Sie außerhalb des Luftraum D.
+.dnoc Bleiben Sie außerhalb des Luftraum C.
+.dr Melden Sie $1.
 .drt Rechtskurve genehmigt.
 .dorbr Kreisen Sie rechts.
 .dorbl Kreisen Sie links.
-.dorbo Kreisen Sie über $uc($1).
+.dorbo Kreisen Sie über $1.
 .dorbn Kreisen Sie nördlich des Platzes.
 .dorbe Kreisen Sie östlich des Platzes.
-.dorbs Kreisen Sie südlich  des Platzes.
+.dorbs Kreisen Sie südlich des Platzes.
 .dorbw Kreisen Sie westlich des Platzes.
-.dtc Fliegen Sie in die Platzrunde Piste $uc($1).
-.dtcr Fliegen Sie in die rechte Platzrunde Piste $uc($1).
-.dtfc Verkehrsinformation, $uc($1), melden Sie in Sicht.
-.dtfc2 Folgen Sie genanntem Verkehr, Nummer 2.
-.ddown Fliegen Sie in den Gegenanflug Piste $uc($1).
-.ddownr Fliegen Sie in den rechten Gegenanflug Piste $uc($1).
+.dtc Fliegen Sie in die Platzrunde Piste $arrrwy.
+.dtcr Fliegen Sie in die rechte Platzrunde Piste $arrrwy.
+.dtfc Verkehrsinformation, $1, melden Sie in Sicht.
+.dtfc2 Nummer 2, folgen Sie genanntem Verkehr.
+.ddown Fliegen Sie in den Gegenanflug Piste $arrrwy.
+.ddownr Fliegen Sie in den rechten Gegenanflug Piste $arrrwy.
 .ddownex Verlängern Sie Gegenanflug, warten Sie für Queranflug.
-.dbase Fliegen Sie in den Queranflug Piste $uc($1).
-.dbaser Fliegen Sie in den rechten Queranflug Piste $uc($1).
-.dfin Fliegen Sie in den Endanflug Piste $uc($1).
-.dwake Achtung Wirbelschleppen.
+.dbase Fliegen Sie in den Queranflug Piste $arrrwy.
+.dbaser Fliegen Sie in den rechten Queranflug Piste $arrrwy.
+.dfin Fliegen Sie in den Endanflug Piste $arrrwy.
+.dwake Vorsicht Wirbelschleppen.
+.dcft Wind $wind, Piste $deprwy, Start frei.
+.dctl Wind $wind, Piste $arrrwy, Landung frei.
+.dctll Wind $wind, Piste $arrrwy, Landung frei, verlassen Sie die Piste nach links.
+.dctlr Wind $wind, Piste $arrrwy, Landung frei, verlassen Sie die Piste nach rechts.
+.dctg Wind $wind, Piste $arrrwy, frei zum Aufsetzen und Durchstarten.
+.dcla Wind $wind, Piste $arrrwy, frei zum Tiefanflug.
+.drdy Melden Sie abflugbereit.
+.dstop Sofort anhalten. Halten Sie Position, Startfreigabe aufgehoben.
+.dga Starten Sie durch.
+.dlu Rollen Sie zum Abflugpunkt Piste $deprwy.
+.dludep Hinter nächster abfliegenden $uc($1), rollen Sie zum Abflugpunkt Piste $deprwy, dahinter.
+.dluarr Hinter nächster anfliegenden $uc($1), rollen Sie zum Abflugpunkt Piste $deprwy, dahinter.
+.dhp Halten Sie Position.
+.dt Rollen Sie zum Rollhalt Piste $deprwy über $uc($1).
+.dts Rollen Sie zu Parkposition $uc($1) über $uc($2).
+.dtgat Rollen Sie zum Abstellplatz der Allgemeinen Luftfahrt über $uc($1).
+.dth Rollen Sie über $uc($1), halten Sie vor $uc($2).
+.dct Rollen Sie weiter.
+.dte Rollen Sie zu Entry $uc($1) über $uc($2).
+.dhs Halten Sie vor $uc($1).
+.dgw Gewähren Sie $1 Vorfahrt.
+.dcol Verlassen der Frequenz genehmigt.
 
 --- HELP ---

```

Comment:
Wie wäre es mit einem Help Command zu 8,33kHz? 

.help833 You have been transferred to an 8.33 kHz frequency. If your aircraft’s radio does not allow you to tune the exact frequency, you can use the .com1 command in your pilot client to manually tune it. Visit vats.im/833 for more details.

Reply:
Done!